### ISSUE TITLE
runDistributable fails in v1.2.0-beta01

### ISSUE THREAD

**Issue by GuilhE** [JetBrains/compose-jb#2329]
> ➜  ./gradlew runDistributable 
> Configuration on demand is an incubating feature.
> Type-safe project accessors is an incubating feature.
> > Task :desktopApp:createRuntimeImage FAILED
> 
> FAILURE: Build failed with an exception.
> 
> * What went wrong:
> A problem was found with the configuration of task ':desktopApp:createRuntimeImage' (type 'AbstractJLinkTask').
>   - In plugin 'org.jetbrains.compose.ComposeCompilerKotlinSupportPlugin' type 'org.jetbrains.compose.desktop.application.tasks.AbstractJLinkTask' property 'javaRuntimePropertiesFile' specifies file '.../desktopApp/build/compose/tmp/checkRuntime/properties.bin' which doesn't exist.
>     
>     Reason: An input file was expected to be present but it doesn't exist.
>     
>     Possible solutions:
>       1. Make sure the file exists before the task is called.
>       2. Make sure that the task which produces the file is declared as an input.
>     
>     Please refer to https://docs.gradle.org/7.5.1/userguide/validation_problems.html#input_file_does_not_exist for more details about this problem.
> 
> * Try:
> > Run with --stacktrace option to get the stack trace.
> > Run with --info or --debug option to get more log output.
> > Run with --scan to get full insights.
> 
> * Get more help at https://help.gradle.org
> 
> Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.
> 
> You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.
> 
> See https://docs.gradle.org/7.5.1/userguide/command_line_interface.html#sec:command_line_warnings
> 
> BUILD FAILED in 1s
> 9 actionable tasks: 1 executed, 3 from cache, 5 up-to-date
> 

Downgrading back to `v1.2.0-alpha01-dev774` works.

**Comment by GuilhE** [comment]
Version `1.2.0-beta02` seems to fix it.

**Comment by GuilhE** [comment]
> Version `1.2.0-beta02` seems to fix it.

False alarm

**Comment by Thomas-Vos** [comment]
Also running into this issue. My build very often fails with this error, but sometimes it works. Would be nice if this could be fixed.

**Comment by cotysaxman** [comment]
I ran into this with 1.2.0 stable. I was able to avoid it by running `:checkRuntime` and then `:createRuntimeImage` with the `--no-build-cache` flag. The error indicated that a required intermediate artifact wasn't being located in the build folder, so it seems to be a bug with the job's caching logic.

Supporting this theory, a clean build would eliminate the needed temporary file, but even after a clean build `:checkRuntime` was using the cached result.
